src/test/java/bio/terra/app/controller/SearchApiControllerTest.java

```diff
@@ -58,6 +62,27 @@
 
   @MockBean private SnapshotService snapshotService;
 
+  @Test
+  public void testSnapshotPreviewById() throws Exception {
+    var id = UUID.randomUUID();
+    var table = "good_table";
+    var limit = 10;
+    var offset = 0;
+    var list = List.of("hello", "world");
+    var result = new SnapshotPreviewModel().result(List.copyOf(list));
+    when(snapshotService.retrievePreview(id, table, limit, offset)).thenReturn(result);
+    mvc.perform(
+            get(GET_PREVIEW_ENDPOINT, id, table)
+                .queryParam("limit", String.valueOf(limit))
+                .queryParam("offset", String.valueOf(offset)))
+        .andExpect(status().isOk())
+        .andExpect(jsonPath("$.result").isArray());
```

Comment:
If you want to, you could assert the content `hello world` too. I don't know the syntax well enough to suggest a fix here though.

Reply:
I tried a couple ways to do this using `is` and `value`, but was not successful.